beer.rb

```diff
@@ -1,11 +1,15 @@
 
+def function_age(user_age)
+	if user_age >= "21"
```

Comment:
I know this worked, but you don't want to use strings with a comparison operator.   If you're doing numeric operations, you should be converting to integers or floats